minetest/minetest
# Add macOS build docs

Download and Build Instructions for MacOS (Intel Hardware), with Git and Homebrew

## Goal of the PR
This PR adds Git/Homebrew download and build instructions for Mac users getting started with Minetest. It mainly incorporates commands from the `macos.yml` workflow file located in the `.github/workflows` directory, which are what I used to create an inital build-from-source.

## How does the PR work?
See above, incorporates existing build commands from the MacOS workflow file.

The MacOS instructions don't include a minimum dependency version tables right now. I found this useful when looking at the GNU/Linux instructions, but don't know how to figure out minimum versions without more knowledge of how Minetest works.

## Does it resolve any reported issue?
N/A

## Does this relate to a goal in [the roadmap](../doc/direction.md)?
Will more accessible build instructions bring more Mac users to Minetest?

## If not a bug fix, why is this PR needed? What usecases does it solve?
Minetest's README currently includes no download/build instructions for MacOS users, and the commands contained in the MacOS Github workflow that _do_ work are hard to find.

## To do

This PR is Ready for Review.

- [ ] Determine minimum version requirements (I have no experience with this, and would welcome advice or contibutions)

## How to test
Ideally, a few (Intel) Mac users would run these instructions top to bottom, and verify that they generate a successful build like they did on my machine.
